applications/picocdn-node-vm/README.md

```diff
@@ -0,0 +1,87 @@
+```text
+SPDX-License-Identifier: Apache-2.0
+Copyright (c) 2021 NetPico LAbs Pvt Ltd.(picoNETS)
+```
+
+
+# **picoCDN Platform Overview**
+picoCDN is a Deep Edge focused Caching and Content Delivery Network (CDN) platform. Our Deep Edge CDN platform performs better than traditional CDNs by virtue of it unique and flexible architecture. We achieve this by bringing your content closer to your users, embedding deep into Telco and ISP networks.  With our Ultra-Low Latency CDN platform, we can deliver a buffer-free experience for Full HD, 4K, AR, VR, and other bandwidth-demanding content. Every edge location of our distribution network is geared to deliver content to users who are just a couple of hops away. We also provide a pre-caching API to push your content to the edge nodes before customers requests reach the node. This version of the platform is designed to help you deploy Edge POPs in various local ISPs, malls, hotels, airports, cellsites or other 5G Multi-Access Edge Computing (MEC) locations running Intel OpenNESS.

```

Comment:
Consider re-writing:  "Avoid using We/Me/You"
picoCDN is a Deep Edge focused Caching and Content Delivery Network (CDN) platform. This Deep Edge CDN platform performs better than other traditional CDNs by virtue of it unique and flexible architecture. This is achieved by bringing the user content closer to user location, embedding deep into Telco and ISP networks.  With this Ultra-Low Latency CDN platform, can deliver a buffer-free experience for Full HD, 4K, AR, VR, and other bandwidth-demanding content. Every edge location of our distribution network is geared to deliver content to users who are just a couple of hops away. This also provide a pre-caching API to push user content to the edge nodes before customers requests reach the node. This version of the platform is designed to help OEM/ODMs deploy Edge POPs in various local ISPs, malls, hotels, airports, cell sites or other 5G Multi-Access Edge Computing (MEC) locations running Intel OpenNESS.

Reply:
Re-written the description block in a neutral perspective, should now avoid the first/second person grammatical constructs.